content/sessions/ai-1026.zh.md

```diff
@@ -11,4 +11,4 @@ Submarine 提供了标准的 Tensorflow、Pytorch、Python 和 XGBoost 等机器
  ### Speakers: 
  <img src="images/speaker/1026.png" width="200" /><br>刘勋: 滴滴, 高级技术专家, Apache Member、Apache 孵化器导师、Apache Submarine PMC Chair、Apache Hadoop & Zeppelin Committer
 
- 
\ No newline at end of file
+ 
```

Comment:
Please add the speaker image and introduction. You also need to update the ai-1026.md at the same time.